server/graphql/resolvers/article/queries.js

```diff
@@ -0,0 +1,94 @@
+const articleQueries = {
+  articles: (_, { search }) => {
+    console.log(search);
+    const articles = [
+      {
+        id: 1,
+        number: 1,
+        content:
+          'Esta prohibido manejar en las siguientes condiciones:\n\t 1. Teniendo los faros rotos\n\t 2. Teniendo las llantas ponchadas\nCualquiera de estos casos no será tolerado!',
+        keywords: [
+          'Lorem',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+          'Ipsum',
+        ],
+        minutesToRead: 5,
```

Comment:
Same goes for this, but this value doesn't affect the fundamental design so I'd say its alright if you leave it